src/io/flutter/editor/GenericColorPicker.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.editor;
+
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * Color picker api with implementations for Android Studio and generic IntelliJ.
+ * <p>
+ * The regular IntelliJ color picker implementation can't be used on Android
+ * Studio and vice versa.
+ */
+public interface GenericColorPicker {
```

Comment:
See src/io/flutter/android/GradleSyncProvider.java for an example of defining an extension point.

Reply:
done